src/github/nag.rs

```diff
@@ -374,6 +441,26 @@ fn evaluate_nags() -> DashResult<()> {
                 continue;
             }
         };
+
+        // Execute FFCP actions:
+        if is_rfc_repo(&issue) {
+            match disp {
+                FcpDisposition::Merge => {
+                    // TODO: This one will require a lot of work to
```

Comment:
Can you start an issue for this? I'm not even sure what the "right" strategy is going to be and it will require some discussion.

Reply:
Done :)